lambda-role-policy.json

```diff
@@ -8,10 +8,12 @@
     },
     {
       "Effect": "Allow",
+      "Action": "ec2:DescribeAddresses",
+      "Resource": "*"
+    },
+    {
       "Action": "cloudwatch:PutMetricData",
       "Resource": "*"
```

Comment:
Am I reading the diff wrong, or is `"Effect": "Allow"` missing for the `cloudwatch:PutMetricData` stanza?